tests/data/placebo/test_sqs_add_statements_policy/sqs.CreateQueue_1.json

```diff
@@ -0,0 +1,17 @@
+{
+    "status_code": 200,
+    "data": {
+        "QueueUrl": "https://queue.amazonaws.com/644160558196/test_sqs_add_statements_policy",
+        "ResponseMetadata": {
```

Comment:
Please make ResponseMetadata an empty dict {} it will reduce bloat in those test folders. I am currently looking into a way to update the recording logic so this update can be automated.